csrc/kernel_cache.cpp

```diff
@@ -225,7 +230,7 @@ std::vector<at::Tensor> FusionExecutorCache::runFusionWithInputs(
 
   // Permute output tensor returned by kernel execution.
   // See Part_3 in Note [ Permutation support in nvfuser ]
-  for (const auto& pair : fusion_->getPermutationOutputMap()) {
+  for (const auto& pair : fusion->getPermutationOutputMap()) {
```

Comment:
No this change is unnecessary; it could be done with either `fusion_` (the symbolic original fusion) or `fusion` (the concretized fusion) since they should have the same permutation map.